cytoscape-edgehandles.js

```diff
@@ -932,6 +936,9 @@ SOFTWARE.
               var isGhost = node.hasClass( 'edgehandles-ghost-node' );
               var noEdge = options().edgeType( source, node ) == null;
 
+              node.addClass('edgehandles-presumptiveTarget');
```

Comment:
Maybe `edgehandles-presumptive-target` so it's consistent with others?

And having this in the readme would be great.  Otherwise, it's hard to find the feature.